src/prefect/client/orion.py

```diff
@@ -1433,7 +1433,7 @@ async def read_flow_run(self, flow_run_id: UUID) -> FlowRun:
             flow_run_id: the flow run ID of interest
 
         Returns:
-            a [Flow Run model][prefect.orion.FlowRun] representation of the flow run
+            a [Flow Run model][prefect.orion.schemas.core.FlowRun] representation of the flow run
```

Comment:
Actually it's `prefect.client.schemas.FlowRun` now; should we even have these links? They're in the type annotation.

Reply:
`prefect.client.schemas.FlowRun` doesn't resolve to a valid link address for mkdocstrings

Reply:
I think we should fix that or remove the link then. There are other cases here where we are incorrectly referencing the orion schemas as well. I'd prefer not mislead users on the return types.